text/0000-grid-inventory.md

```diff
@@ -0,0 +1,126 @@
+Inventory component, and is inspired by GS1’s EDI Inventory Report - Business
+Message Standard. GS1 is a widely used data standard in enterprises and, given
+that positioning and familiarity, Grid support feels natural. Additional
+implementations of Grid Inventory may derive from or extend this
+implementation.
+
+Grid Inventory leverages other extensible Hyperledger Grid components
+including: Grid Product, and Grid Location. Grid Inventory aims to provide
+current inventory quantity & status visibility for each Grid Product and Grid
+Location defined within the Grid Inventory smart contract between parties, as
+well as a summary of relevant inventory events that occurred within a
+time-range.  It includes transactions for generating summary inventory report
+information from external systems of record and recording the results in state.
+
+Grid Inventory also aims to serve as a foundation for future potential Grid
+capabilities that may require inventory related information, such as purchasing
+and logistics.
+
+# Motivation
+[motivation]: #motivation
+
+The Grid Inventory implementation is designed to enable shared visibility of
+summary inventory information for any defined Grid Product and Grid Location
+between participants.  Management of inventory levels is a common concept
+within supply chain solutions and is a key primary business activity related
+to all purchases of physical goods from suppliers, as well as all sales of
+physical goods to customers.  Management of inventory levels is typically
+coordinated very manually and very independently between parties across siloed
+systems and processes.  This siloed and manual coordination of inventory
+between parties can lead to substantial pain-points, including:
+inefficiencies, waste, and poor coordination of inventory levels across
+the supply chain.
+
+This Grid Inventory concept aims to solve these issues by offering a common
+industry solution for vertically integrating inventory information between
+parties existing systems of record.
+
+The Grid Inventory design addresses several use cases, including:
+
+- View Current Inventory Status Data: Viewing current summary inventory
+  quantities and inventory statuses between trading partners for defined Grid
+  Products and Grid Locations.
+
+- View Historical Inventory Activity Data: Viewing historical inventory
+  activity between trading partners for defined Grid Products and Grid
+  Locations.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+TBD - based on discussion.  We are exploring designs based on:
+
+ - GS1’s EDI Inventory Report - Business Message Standard.
+ - GS1's X12 EDI Standards.
+ - GS1's EPCIS Standard.
+ - Other standards as relevant.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+TBD - based on discussion.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+By leveraging GS1’s EDI Inventory Report standards, only a summary view of
```

Comment:
Please add a link to the specific GS1 spec being referenced ... they can change

Reply:
Thank you Jason - will do.  I am actually currently looking at the X12 EDI standards from GS1, as those are more commonly leveraged in North America.  The above inventory report standard is actually more catered to outside of North America.

Once i have enough learnings to make the update to this RFC draft, we can definitely include the actual link to the GS1 spec.